hack/lib/init.sh

```diff
@@ -37,6 +37,7 @@ route/v1 \
 samples/v1 \
 security/v1 \
 securityinternal/v1 \
+platform/v1alpha1 \
```

Comment:
This list is nearly alphabetical order, can you add to the right place near the project entry instead please